### PR TITLE
9.8.0 - Readme, Changelog and galaxy.yml updates

### PR DESCRIPTION
**Pull request by rajshekarp87**
# Description
A few sentences describing the overall goals of the pull request's commits.

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| |

# ISSUE TYPE

- Bugfix Pull Request
- Docs Pull Request
- Feature Pull Request
- Test Pull Request

##### OUTPUT

```paste below

```
##### ADDITIONAL INFORMATION

```paste below

```
# Checklist:

- [ ] I have performed a self-review of my own code to ensure there are no formatting, vetting, linting, or security issues
- [ ] I have verified that new and existing unit tests pass locally with my changes
- [ ] I have not allowed coverage numbers to degenerate
- [ ] I have maintained at least 90% code coverage
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have maintained backward compatibility

### CODE REVIEW

**Comment by ansible-collections-svc** [comment]
Can one of the admins verify this patch?

**Review by sachin-apa — DISMISSED** [review]

File: CHANGELOG.rst [hunk 1]
```diff
@@ -4,6 +4,45 @@ Dell OpenManage Ansible Modules Release Notes
...
+-------------
+
+- omevv_vcenter_info - This module allows to retrieve vCenter information.
+- omevv_firmware_repository_profile_info - This module allows to retrieve firmware repository profile information.
+- omevv_firmware_repository_profile - This module allows to manage firmware repository profile.
+
+Minor Changes
+-------------
+
+- idrac_system_info - This module is enhanced to support iDRAC10 and OMSDK dependency is removed.
```
Comment (Saksham-Nautiyal) [inline CHANGELOG.rst]: This needs to be edited

File: CHANGELOG.rst [hunk 2]
```diff
@@ -4,6 +4,45 @@ Dell OpenManage Ansible Modules Release Notes
...
+Major Changes
+-------------
+
+- omevv_vcenter_info - This module allows to retrieve vCenter information.
+- omevv_firmware_repository_profile_info - This module allows to retrieve firmware repository profile information.
+- omevv_firmware_repository_profile - This module allows to manage firmware repository profile.
+
+Minor Changes
+-------------
+
```
Comment (singh-lovepreet1) [inline CHANGELOG.rst:28]: Please remove this.

File: CHANGELOG.rst (hunk 1 above)
Comment (singh-lovepreet1) [inline CHANGELOG.rst]: Please remove this as we are not delivering this one.

**Review by sachin-apa — APPROVED** [review]

**Review by Saksham-Nautiyal — APPROVED** [review]

**Review by singh-lovepreet1 — APPROVED** [review]

File: CHANGELOG.rst (hunk 1 above)
Comment (rajshekarp87) [reply CHANGELOG.rst]: Done

**Comment by sachin-apa** [comment]
run e2e

**Pull request merged**